src/Stripe.net/Services/Disputes/DisputeCloseOptions.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Stripe
```

Comment:
Try to separate your PRs by conceptual changes. Adding those new classes is wildly different from removing `tax_percent` and it makes it hard to pinpoint the root cause of the changes